dbctx/models.go

```diff
@@ -10,25 +10,25 @@ type (
 	// Soup holds soup metadata
 	Soup struct {
 		gorm.Model
-		Name          string `gorm:"unique_index"`
-		SoupOfTheDays []SoupOfTheDay
+		Name          *string `gorm:"unique_index"`
```

Comment:
What is the reasoning behind changing this to pointers?

Reply:
Had a bug where no comment was set to the empty string - this will make it null instaed.

Reply:
So basically it's saying null is allowed.